doc/release-notes.rst

```diff
@@ -2,6 +2,72 @@
  Release Notes
 ===============
 
+v0.87.2 Giant
+=============
+
+Notable Changes
+---------------
+* allow -L to disable lttng. Enable it by default (Alfredo Deza)
+* mon: MDSMonitor: wait for osdmon to be writable when requesting proposal (#9794 Joao Eduardo Luis)
+* mon: MDSMonitor: have management_command() returning int instead of bool (Joao Eduardo Luis)
```

Comment:
Can be skipped because it's part of the same pull request as #9794 https://github.com/ceph/ceph/pull/4258